app/cypress/support/commands.js

```diff
@@ -147,7 +147,7 @@ Cypress.Commands.add(
     cy.contains(opsTeamPrimary).click();
 
     // FIXME: adding project managers does not trigger the saving indicator
```

Comment:
Delete comment? If the line 150 assertion works, then I think the saving indicator has to be working?